src/cli.js

```diff
@@ -31,7 +31,7 @@ function printErrorIter(error, logger) {
 
 process.on('unhandledRejection', (error) => {
   printErrorIter(error, new Logger());
-  process.exit(1);
+  throw new Error(error);
```

Comment:
I think we can simplify this a bit, since we already have an Error instance
```suggestion
  throw error;
```

Reply:
You're right 👍